examples/light-switch-app/light-switch-common/light-switch-app.matter

```diff
@@ -1093,6 +1149,91 @@ client cluster Scenes = 5 {
   readonly attribute bitmap8 nameSupport = 4;
   readonly global attribute attrib_id attributeList[] = 65531;
   readonly global attribute int16u clusterRevision = 65533;
+
+  request struct AddSceneRequest {
```

Comment:
And this scene stuff?

Reply:
No, I did not add these (AddSceneRequest and AddGroupRequest) on purpose. Is it possible that ZAP made this decision based on the Access Control Cluster being turned ON?

Reply:
Not that I know of, but I am not a ZAP expert. Maybe those apps just haven't been ZAP regenerated lately? (But I thought CI checked for that?)

Reply:
ZAP Regeneration post-rebase fixed it.

Reply:
Seems pretty finicky- every other regen adds it back, while the other removes it.